tests/integrations_setup/install_cspm_gcp_integration.py

```diff
@@ -48,6 +48,19 @@
 }
 
 
+def read_json_file(file_path):
+    """Reads a json file and returns its content"""
+    try:
+        with open(file_path, "r") as json_file:
+            return json_file.read()
+    except FileNotFoundError:
+        logger.error(f"Error: File '{file_path}' not found.")
+        sys.exit(1)
+    except IOError as e:
+        logger.error(f"Error reading file '{file_path}': {e}")
+        sys.exit(1)
+
```

Comment:
@orouz, I think there's a bit of confusion here. The function `read_json_file()` and the variable `json_file` might suggest that they specifically handle JSON files, but they're actually generic and can read any type of file. For reading JSON files, we have a separate function defined [here](https://github.com/elastic/cloudbeat/blob/main/tests/fleet_api/utils.py#L26).

Reply:
@gurevichdmitry i would prefer using `read_json` but it returns an empty dict when it fails (also missing decode exception) and i don't want to check for an empty dict in my use case and then exit, as it's kinda dirty. i'd prefer updating `read_json` to remove the `try` and let consumers do it and act on errors. in my case, i would `sys.exit(1)` if reading the file failed. can we do the same in [`load_data()`](https://github.com/elastic/cloudbeat/blob/e8309ef67d6bb49119a9464ff00ae0b9bf815a44/tests/integrations_setup/install_d4c_integration.py#L47-L48) ?

Reply:
@orouz considering that `read_json` is only used in `load_data`, we can modify `read_json` to exit on error. This approach fulfills your requirement, and the reuse will be valid in your case.